config.xhgui.yaml

```diff
@@ -0,0 +1,11 @@
+#ddev-generated
+hooks:
+  post-start:
+  # Create a new database called "xhgui"
+  - exec: |
+      if [[ "$DDEV_DATABASE_FAMILY" == "postgres" ]]; then
```

Comment:
IMO there's nothing wrong with this approach, but would `ddev import-db --database` be simpler? It would just have to choose a reasonable sql dump file for each of these.

Reply:
Importing a DB would work, but that also means we need to maintain a DB file.
This approach feels more future-proof to me. 
Happy to reconsider if you feel strongly though.

Reply:
This works!